ext/reflection/php_reflection.stub.php

```diff
@@ -261,6 +264,15 @@ public function getMethod(string $name): ReflectionMethod {}
     /** @tentative-return-type */
     public function getMethods(?int $filter = null): array {}
 
+    /** @tentative-return-type */
+    public function hasOperator(string $name): bool {}
+
+    /** @tentative-return-type */
+    public function getOperator(string $name): ReflectionMethod {}
+
+    /** @tentative-return-type */
+    public function getOperators(?int $filter = null): array {}
+
```

Comment:
These are new functions, therefore they can just have the return type be normal and not be tentative IMHO.